### PR TITLE
Level 1 Return Type Recipes

### PR DESCRIPTION
**Pull request by tylercwiltz**
3 recipes on return types.  Solutions are in a branch called L1M1-ReturnTypeRecipesSolutions.

### CODE REVIEW

File: Level1-Module0/src/_02_return_types/_3_too_many_shapes/TooManyShapes.java [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+			//6. Use drawPolygons() to draw your shape
+		
+		//7. If the shape CANNOT be drawn 
+			
+			//8. Call notEnoughSides() and print out what is returned 
+		
+	}
+	
+	static int calculateTurnAngle(int numSides) {
+		int angle = 180-(((numSides-2)*180)/numSides);
```
Comment (dencee) [inline Level1-Module0/src/_02_return_types/_3_too_many_shapes/TooManyShapes.java:34]: Still works, but how did you get this equation? If you simplify it, it's just angle = 360 / numSides
angle = 180 - (180*n - 360) / n
angle = 180 - 180*n/n + 360/n
angle = 180 - 180 + 360/n
angle = 360/n

File: Level1-Module0/src/_02_return_types/_3_too_many_shapes/TooManyShapes.java [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+			int y = rand.nextInt(600);
+			int angle = rand.nextInt(360);
+
+			rob.setX(x);
+			rob.setY(y);
+			rob.setAngle(angle);
+			rob.setRandomPenColor();
+			
+	    	for(int j = 0; j < numSides; j++) {
+	    		rob.move(dist);
```
Comment (dencee) [inline Level1-Module0/src/_02_return_types/_3_too_many_shapes/TooManyShapes.java:55]: mixed spaces and tabs here

**Review by dencee — CHANGES_REQUESTED** [review]
Rest looks good!

File: Level1-Module0/src/_02_return_types/_3_too_many_shapes/TooManyShapes.java (hunk 1 above)
Comment (tylercwiltz) [reply Level1-Module0/src/_02_return_types/_3_too_many_shapes/TooManyShapes.java]: Nice catch.  Idk why I had it the other way.  This was an older recipe I wrote that I modified, but I can't remember if there was a reason I did it that way.

File: Level1-Module0/src/_02_return_types/_3_too_many_shapes/TooManyShapes.java (hunk 2 above)
Comment (tylercwiltz) [reply Level1-Module0/src/_02_return_types/_3_too_many_shapes/TooManyShapes.java]: I checked them and they all seem to be tabs.  I tabbed them again just to be sure, but it looks all good on my end. So let me know if it's still wonky.

**Review by dencee — APPROVED** [review]

**Pull request merged**